Matchfile

```diff
@@ -0,0 +1,13 @@
+git_url("https://github.com/jhass/insporation-certificates")
```

Comment:
Maybe better `git@github.com:jhass/insporation-certificates.git`?

Reply:
Don't know if this 'better' or follows a recommended way - the URL was added by a generator. Change it